.eslintrc.js

```diff
@@ -1,21 +1,36 @@
 module.exports = {
   root: true,
+  parser: '@typescript-eslint/parser',
   extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
-  env: {
-    commonjs: true,
-  },
+
   overrides: [
     {
```

Comment:
We should use the module template .eslint config, which has this override option for typescript:

```typescript
{
  files: ['*.ts'],
  extends: ['@metamask/eslint-config-typescript'],
}
```